src/test/scala/mesosphere/marathon/core/appinfo/TaskCountsTest.scala

```diff
@@ -182,7 +184,7 @@ class Fixture {
       startedAt = None,
       mesosStatus = None
     ),
-    Task.NoNetworking
+    Seq.empty
```

Comment:
as you have realized by now, I really like named arguments in these cases (Seq.empty doesn't give you a hint what the parameter is).